pages/admin/request/create-new.tsx

```diff
@@ -354,8 +353,8 @@ export default function CreateNew() {
           usesAccessibleConvertedVan: additionalQuestions.usesAccessibleConvertedVan,
           requiresWiderParkingSpace: additionalQuestions.requiresWiderParkingSpace,
 
-          ...paymentDetails,
-          paymentMethod: paymentDetails.paymentMethod,
+          ...values.paymentInformation,
+          paymentMethod: values.paymentInformation.paymentMethod,
```

Comment:
won't need to explicitly set paymentMethod once values are validated with above fix.

Reply:
Addressed but relates to https://github.com/uwblueprint/richmond-centre-for-disability/pull/163#discussion_r841191472